packages/strapi-admin/services/role.js

```diff
@@ -0,0 +1,19 @@
+module.exports = {
```

Comment:
So we are consistent we the rest of the feature can you create your functions with function expressions and export them after.


I think we should rename fetch to findOne is fetch is a legacy term coming from bookshelf queries